InvenTree/plugin/registry.py

```diff
@@ -187,6 +187,53 @@ def reload_plugins(self, full_reload: bool = False):
 
         logger.info('Finished reloading plugins')
 
+    def plugin_dirs(self):
```

Comment:
Suggestion: @SchrodingersGat should we maybe move this out of the class? Would keep the functions cleaner.

Reply:
Happy to, I moved it out of settings.py because that requires a full reload to update (and thus can't be unit tested).

Still having trouble with testing plugins from external dirs.

Reply:
@SchrodingersGat a lot of plugin stuff needs reloads - you can easily provoke that with `registry.reload_plugins()`.

Reply:
@matmair except that the only place where the environment variables are inspected is in `settings.py` - so you can't "mock" environment vars and then call `reload_plugins()` - because the environment vars are not re-processed. Does this make sense?